app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventDetailActivity.kt

```diff
@@ -335,6 +335,8 @@ class ProgramEventDetailActivity :
                             programStageUid = it,
                         )
                     }
+                } else {
+                    binding.addEventButton.isEnabled = true
```

Comment:
do you think that could be more readable to refactor this line and the 321 to a method with a parameter like `addEventEnabled(enable: Boolean)` ?

Reply:
Hadn't seen this, I'll update it